client/utils/get-error-field.js

```diff
@@ -1,11 +1,13 @@
 import React from "react";
 
-const getErrorField = (errors, key, nonField = false, className = "") =>
-  errors[key] ? (
-    <div className={!nonField ? `error ${className}` : "error non-field"}>
+const getErrorField = (errors, key) => {
```

Comment:
if we use `key=null`, then `getErrorField(errors, null)` can become `getErrorField(errors)`.
At this point we can wonder, why name it `getErrorField` if it can also show non field errors? Then we could simply name it `getError`, what do you think?

Reply:
Yes